examples/sink/async_log/example.py

```diff
@@ -4,27 +4,28 @@
 
 from pynumaflow.sinker import Datum, Responses, Response, Sinker
 from pynumaflow.sinker import SinkAsyncServer
+from pynumaflow._constants import _LOGGER
```

Comment:
Where is this logger coming from? `logging` module is a singleton class and should be defined for each file.

`logging.basicConfig(level=logging.DEBUG)  # will make sure that you see in console 
_LOGGER = logging.getlogger(__name__)`

Also, let's not import from a private file outside of the pynumaflow module: `_constants`

Reply:
Updated! Will create a separate issue for the overall logging as discussed.